radio/src/gui/horus/widgets/outputs.cpp

```diff
@@ -29,33 +33,60 @@ class OutputsWidget: public Widget
     }
 
     virtual void refresh();
+    
+    uint8_t drawChannels(uint16_t x, uint16_t y, uint16_t w, uint16_t h, uint8_t firstChan)
+    {
+      char chanString[] = "CH32";
+      uint8_t lastChan = firstChan + floor(h / ROW_HEIGHT);
```

Comment:
Avoid working with floats whenever it's possible, here ```floor``` is not useful, the result will be always ```h / ROW_HEIGHT``` truncated.